### PR TITLE
Tysonn hometown

### PR DESCRIPTION
**Pull request by ghost**
Closes #17

### CODE REVIEW

**Comment by tysonn** [comment]
Closes #17

**Comment by ghost** [comment]
@kriscrider - Look at my new article

**Review by kriscrider — CHANGES_REQUESTED** [review]
What about the snow?

File: tysonn-ketchikan.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Ketchikan, AK
+
+## Places to go
+
+- Creek Street
+- Totem Bight State Park
+
+## Food to eat
+
+- Grilled Salmon
```
Comment (EShaheenGit) [inline tysonn-ketchikan.md]: @tysonn we have Salmon here!

**Review by EShaheenGit — CHANGES_REQUESTED** [review]
find something other than Salmon

File: tysonn-ketchikan.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+## Ketchikan, AK
+
+## Places to go
+
```
Comment (Monasr) [inline tysonn-ketchikan.md:4]: Looks good to me

**Pull request merged**